compressed_image_transport/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 cmake_minimum_required(VERSION 3.0.2)
 project(compressed_image_transport)
 
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/modules" ${CMAKE_MODULE_PATH})
+
 find_package(OpenCV REQUIRED)
+find_package(TurboJPEG REQUIRED)
 find_package(catkin REQUIRED COMPONENTS cv_bridge dynamic_reconfigure image_transport)
```

Comment:
```suggestion
find_package(OpenCV REQUIRED)
find_package(catkin REQUIRED COMPONENTS cv_bridge dynamic_reconfigure image_transport)

find_package(PkgConfig REQUIRED)
pkg_check_modules(TurboJPEG REQUIRED libturbojpeg)
```

Reply:
Ubuntu, Fedora, Debian, Arch and probably all other distros provide a pkgconfig file, so let's use it.

Reply:
Ubuntu 16.04 does not provide a pkgconfig file. But I guess since Kinetic is EOL we don't care about 16.04.